doc/radosgw/placement.rst

```diff
@@ -147,6 +147,12 @@ Then provide the zone placement info for that storage class:
         --data-pool default.rgw.cold.data \
         --compression lz4
 
+Finally, restart the Ceph Object Gateway:
+
+::
+
+  # systemctl restart ceph-radosgw@rgw.`hostname -s`
+
```

Comment:
earlier in the doc, there's a note about this:

> Note: If you have not done any previous Multisite Configuration, a default zone and zonegroup are created for you, and changes to the zone/zonegroup will not take effect until the Ceph Object Gateways are restarted. If you have created a realm for multisite, the zone/zonegroup changes will take effect once the changes are committed with radosgw-admin period update --commit.

we might want to move that note down here and possibly rephrase it as the final step

Reply:
ping @ofriedma